ystia/yorc
# Feature/brbdcf 1302 gcp compute instance deployment

# Pull Request description

## Description of the change

Associated to Yorc Alien4Cloud plugin Pulll Request https://github.com/ystia/yorc-a4c-plugin/pull/26

### What I did

Added a yorc.nodes.google.Compute type and code to provision a Google Compute Instance through its Terraform provider.

Updated the documentation at doc/configuration.rst and doc/infrastructures.rst

### How to verify it

See doc/configuration.rst in to configure a google infrastructure.
Then see the Yorc A4C plugin associated Pull Request to configure a Google Cloud location.
A Tosca sample is also provided in the Yorc A4C plugin associated Pull Request at tosca-samples/org/ystia/yorc/samples/vision
Follow the instructions in the README provided with this sample to deploy the sample and check the application works
